modules/issue_tracker/php/edit.class.inc

```diff
@@ -429,7 +452,7 @@ class Edit extends \NDB_Page implements ETagCalculator
 
         $issueID = intval($values['issueID']);
 
-        $issueValues['lastUpdatedBy'] = $user->getUsername();
+        $issueValues['lastUpdatedBy'] = $user->getUserName();
```

Comment:
since it is assigned in line 483 I think this input in 455 is not longer necessary .

Reply:
It will be because it is used in line 473 (update on existing issue case) and 477 (new issue case).
This needed to be the username (userID) for the `issues` table. The value is just changed for the full name after database insert/update (line 483).

Reply:
I see, yes, it's a bit confusing the way the code is wrote but I think this precedes this PR.  For example **$historyValues** is assigned in 470 the used in 486, passing the DB updated in between without using the value :) 

The PR is fine !